### ISSUE TITLE
weapon movement between hands

### ISSUE THREAD

**Issue by fartwhif** [ACEmulator/ACE#744]
### Bug Report or Feature Request (mark with an `x`)
```
- [ X ] bug report -> please search issues before submitting
- [ ] feature request
```

### Repro steps.
1. equip swords in both hands.
2. try to drag a sword from one hand to the other.
### The log given by the failure.

sometimes:
ERROR: Player_Inventory PickupItemWithNetworking item == null

sometimes: 
ERROR: Player_Inventory PickupItemWithNetworking picking up items from world containers side pack WIP

result:  with either log message, the item move/use state is permanently active for the remainder of the session, all equipment changes are denied "You can only move or use one item at a time" and SHIFT+ESC doesn't work.  Relaunch of the client is necessary to synchronize.

### Desired functionality.
the weapons are swapped

### Mention any other details that might be useful.
/weapons command to generate the items

**Comment by gmriggs** [comment]
This has been addressed in https://github.com/ACEmulator/ACE/pull/956

Open questions:

 - When dragging main hand weapon to empty offhand slot, were the wield requirements re-checked, and were the weapon item spells re-applied?

- With 2 weapons equipped, when dragging 1 wielded weapon to the other slot, did retail actually swap the weapons inline? This would seem natural, but I am seeing some weird messages being sent from the client. The client seems to be sending 2 duplicate messages to 'PutItemInContainer' for the item already in the destination slot.

**Comment by Mag-nus** [comment]
Should be fixed in #1197
